pkg/core/blockchain.go

```diff
@@ -384,14 +367,14 @@ func (bc *Blockchain) storeBlock(block *Block) error {
 				return fmt.Errorf("could not find previous TX: %s", prevHash)
 			}
 			for _, input := range inputs {
-				unspent, err := chainState.unspentCoins.getAndUpdate(bc.store, input.PrevHash)
+				unspent, err := chainState.unspentCoins.getAndUpdate(chainState.store, input.PrevHash)
```

Comment:
That doesn't look like `nit refactoring`, it's more like a part of d4df49eb163c745e6096331572acb4f55c65f8c1.